lab_1/main.py

```diff
@@ -214,7 +219,12 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    with open(path_to_file, "w") as f:
+        for item in content:
+            value = ' '.join(item)
+            value += '\n'
+            f.write(value)
```

Comment:
can you put all values into one text
and then write this text to a file
do not write to file at each iteration